setup-stage1.sh

```diff
@@ -3,25 +3,33 @@
 . helpers/error.sh
 . helpers/prompt.sh
 . helpers/funcs.sh
+. helpers/ensure-aur-xlcore.sh
 
+. config/xlcore.sh
 # Determine where the user wants to install the tools
 . config/ffxiv-tools-location.sh
 
+echo "Checking installation..."
+CHECK_FOR_FLATPAK
+
+echo
+echo "Detected environment: $PLATFORM"
+echo
+
 echo "Setting up the FFXIV Environment scripts."
 echo
-echo "This script will require you to open the FFXIV launcher from Lutris as if you were going to play the game normally"
+echo "This script will require you to launch the game from XIVLauncher Core."
 echo
 
 while true; do
-    GET_NEWEST_PID "FFXIV_PID" '[A-Z]:\\.*\\XIVLauncher.exe(?: --steamticket=[^\s]+)?$'; PID_SUCCESS=$?
+    GET_NEWEST_PID "FFXIV_PID" '[A-Z]:\\.*\\ffxiv_dx11.exe(?: --steamticket=[^\s]+)?'; PID_SUCCESS=$?
     [[ "$PID_SUCCESS" -eq 0 ]] && break
-    [[ -z "$XIVLAUNCHER_WARN" ]] && { warn "Please open the XIVLauncher Launcher. Checking for process \"XIVLauncher.exe\"..."; XIVLAUNCHER_WARN="Y"; }
+    [[ -z "$XIVLAUNCHER_WARN" ]] && { warn "Please open the game. looking for process \"ffxiv_dx11.exe\""; XIVLAUNCHER_WARN="Y"; }
     sleep 1
 done
 
 success "FFXIV Launcher PID found! ($FFXIV_PID)"
 echo "Building environment information based on FFXIV Launcher env..."
-
```

Comment:
The array of required env vars below this should probably be updated/replaced with vars that are relevant to XLCore instead, for sanity's sake. Doesn't need to be in this PR but either a followup PR or an issue opened for it would be appreciated.

Reply:
I'll make an issue for that, since I don't have a sub right now to actually test the environment variables.